tests/test_maskedImage.py

```diff
@@ -817,6 +817,9 @@ def testString(self):
         self.assertIn("image=", str(image))
         self.assertIn("mask=", str(image))
         self.assertIn("variance=", str(image))
+        self.assertIn(str(np.zeros((100, 100), dtype=image.image.dtype)), str(image))
+        self.assertIn(str(np.zeros((100, 100), dtype=image.mask.dtype)), str(image))
+        self.assertIn(str(np.zeros((100, 100), dtype=image.variance.dtype)), str(image))
```

Comment:
I suspect that, as written, these three assertions are largely redundant (e.g., the image representation alone could cause all three to pass). Consider combining with the "image=" tests just above.

Reply:
I think that's not quite true, because the mask will be integral while the image and variance are floats. I do agree that the latter two are redundant, but I think it's clearer to include all thee.

Reply:
That depends on the type of `MaskedImage`; integer pixels *are* allowed. 🙂